K8s-deployment/Charts/auditing-server/templates/ingress/ingress.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if .Values.ingress.enabled }}
```

Comment:
Ingress is not needed. There is web api endpoint exposed or api server. The auditing server is internal server with no apis exposed